python/ray/autoscaler/kubernetes/operator_configs/helm/ray/values.yaml

```diff
@@ -0,0 +1,56 @@
+# Default values for Ray.
+
+# RayCluster settings:
+
+# Ray image to use for the head and workers of this Ray cluster.
+image: rayproject/ray:1.3.0
+# Pod type for the Ray head node (as configured below).
+headPodType: ray-head-type
+podTypes:
+    # Since we set headPodType: head-node, the Ray head pod will use the configuration
+    # defined in this entry of podTypes:
+  - name: ray-head-type
+    # No worker pods of this pod type. Thus, we set minWorkers and maxWorkers to 0.
+    minWorkers: 0
+    maxWorkers: 0
+    # Number of CPUs used by this pod type.
+    # (Used for both requests and limits. Must be an integer, as Ray does not support fractional CPUs.)
+    numCPU: 1
+    # Memory used by this Pod type.
+    # (Used for both requests and limits.)
+    memory: 512Mi
+    # Number of NVIDIA GPUs used by this pod type.
+    # (Optional, requires GPU nodes with appropriate setup. See https://docs.ray.io/en/master/cluster/kubernetes-gpu.html)
+    numGPU: 0
+  - name: ray-worker-type
+    # Minimum number of Ray workers of this pod type to keep running.
+    minWorkers: 2
+    # Maximum number of Ray workers of this pod type to which Ray will scale.
+    maxWorkers: 3
+    # Memory used by this Pod type.
+    # (Used for both requests and limits.)
+    memory: 512Mi
+    # Number of CPUs used by this pod type.
+    # (Used for both requests and limits. Must be an integer, as Ray does not support fractional CPUs.)
+    numCPU: 1
+    # Number of NVIDIA GPUs used by this pod type.
+    # (Optional, requires GPU nodes with appropriate setup. See https://docs.ray.io/en/master/cluster/kubernetes-gpu.html)
+    numGPU: 0
```

Comment:
You may also want to allow the user to specify the GPU sku here as well. For example:

```
nodeSelector:
    cloud.google.com/gke-accelerator: nvidia-tesla-v100
```

See https://cloud.google.com/kubernetes-engine/docs/how-to/gpus#multiple_gpus